.coderabbit.yaml

```
@@ -0,0 +1,15 @@
+# .coderabbit.yaml
+language: "ko-KR"
+early_access: false
+reviews:
+  profile: "chill" 
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)